addons/l10n_eg_edi_eta/models/account_move.py

```diff
@@ -98,3 +98,11 @@ def _l10n_eg_edi_exchange_currency_rate(self):
         from_currency = self.currency_id
         to_currency = self.company_id.currency_id
         return abs(self.invoice_line_ids[0].balance / self.invoice_line_ids[0].amount_currency) if from_currency != to_currency and self.invoice_line_ids else 1.0
+
+    def _retry_edi_documents_error_hook(self):
+        for edi_doc in self.edi_document_ids.filtered(lambda d: d.edi_format_id.code == 'eg_eta'):
+            if edi_doc.error:
+                error = json.loads(edi_doc.error)
```

Comment:
How is an HTML field a json?  (maybe it is like that, but...) @poma-odoo

Reply:
@jco-odoo  thanks, fixed. 
I am not familiar with this l10n, and I have no way to test it, I hope it works now